### ISSUE TITLE
new data source for routes

### ISSUE THREAD

**Issue by michaelso** [jpatokal/openflights#828]
@jpatokal I learnt that the old source you guys rely on isn't updating anymore (#330 , #561 )
As I would like to build some route searching tools for my projects, I would to ask if you guys find any new data source? 
Or can share any pointers?

I can help updating the route db as I would like to use it in my projects.
Thank you!

**Comment by jpatokal** [comment]
No, we have not found a new source yet.  I'm working on deriving our own data, but it's slow going and there is no ETA.

**Comment by michaelso** [comment]
I see. How are you guys doing it? Parsing airline timetable manually?
Just see if I can help in building up the db. Guess you guys would like to have some volunteers.
